js/login-module.js

```diff
@@ -0,0 +1,90 @@
+function showPassword() {
+  const togglePassword = document.querySelector("#togglePassword");
+  const password = document.querySelector("#password");
+  const type =
+    password.getAttribute("type") === "password" ? "text" : "password";
+  password.setAttribute("type", type);
+
+  const showIcon = togglePassword.querySelector(".show-icon");
+  const hideIcon = togglePassword.querySelector(".hide-icon");
+  showIcon.classList.toggle("hidden");
+  hideIcon.classList.toggle("hidden");
+}
+
+function validateInputs() {
+  const usernameInput = document.getElementById("username");
+  const passwordInput = document.getElementById("password");
+  const loginButton = document.getElementById("loginBtn");
+
+  if (usernameInput.value && passwordInput.value) {
+    loginButton.removeAttribute("disabled");
+  } else {
+    loginButton.setAttribute("disabled", true);
+  }
+}
+
+//////////////////////////////////////////
+
+function login(showhide) {
+  if (showhide === "show") {
+    document.querySelector(".main-container").style.visibility = "visible";
+  } else if (showhide === "hide") {
+    document.querySelector(".main-container").style.visibility = "hidden";
+  }
+
+  // const forgotPasswordLink = document.createElement("a");
+  // forgotPasswordLink.href = "https://account.wire.com/forgot/";
+  // forgotPasswordLink.innerText = "Forgot password?";
+  // document.querySelector(".main-container").appendChild(forgotPasswordLink);
+}
```

Comment:
will this be used in the future?

Reply:
I don't think so. Didn't want to delete it in case it is used somewhere else. My plan was to reorganize the js files in the later PRs